cmd/csi-snapshotter/main.go

```diff
@@ -98,10 +103,18 @@ var (
 )
 
 func main() {
+	flag.Var(utilflag.NewMapStringBool(&featureGates), "feature-gates", "A set of key=value pairs that describe feature gates for alpha/experimental features. "+
+		"Options are:\n"+strings.Join(utilfeature.DefaultFeatureGate.KnownFeatures(), "\n"))
+
 	klog.InitFlags(nil)
+	flag.CommandLine.AddGoFlagSet(goflag.CommandLine)
```

Comment:
see previous point

Reply:
I tried using flag package for this, but it shows error

Reply:
Which error are you referring to? Can you share it here?